cmd/gpu_plugin/README.md

```diff
@@ -74,24 +76,31 @@ $ grep i915 /sys/class/drm/card?/device/uevent
 
 #### Drivers for discrete GPUs
 
+> **Note**: Kernel (on host) and user-space drivers (in containers)
+> should be installed from the same repository as there are some
+> differences between DKMS and upstream GPU driver uAPI.
+
 ##### Kernel driver
 
 ###### Intel DKMS packages
 
-`i915` GPU driver DKMS[^dkms] package is recommended until Intel
-discrete GPU support in upstream is complete.  It can be installed
-from Intel package repositories for a subset of older kernel versions
-used in enterprise / LTS distributions:
+`i915` GPU driver DKMS[^dkms] package is recommended for Intel
+discrete GPUs, until their support in upstream is complete.  DKMS
+package(s) can be installed from Intel package repositories for a
+subset of older kernel versions used in enterprise / LTS
+distributions:
 https://dgpu-docs.intel.com/installation-guides/index.html
 
 [^dkms]: [intel-gpu-i915-backports](https://github.com/intel-gpu/intel-gpu-i915-backports).
 
 ###### Upstream kernel
 
-With upstream 6.x kernels, discrete GPU support needs to be enabled using
-kernel `i915.force_probe=<PCI_ID>` command line option until relevant kernel
-driver features have been completed also in upstream:
-https://www.kernel.org/doc/html/latest/gpu/rfc/index.html
+Upstream Linux kernel 6.2 or newer is needed for Intel discrete GPU
+support.  For now, upstream kernel is still missing support few of the
```

Comment:
```suggestion
support.  For now, upstream kernel is still missing support for a few of the
```

Reply:
Thanks, amended the commit and pushed it.